web/src/pages/User/components/LoginMethodPassword.tsx

```diff
@@ -105,7 +105,6 @@ const LoginMethodPassword: UserModule.LoginMethod = {
         const result = await request('/apisix/admin/user/login', {
           method: 'POST',
           requestType: 'json',
-          prefix: '',
```

Comment:
@zzzdong Hi, if we have this prefix, what will happen?

Reply:
It will replace the global [`RequestConfig.prefix`](https://github.com/apache/apisix-dashboard/blob/88f323285f2bdbafca027da9aef357b2e3eb4008/web/src/app.tsx#L59)，which unexpected when use dashboard under subpath.

Reply:
ok!

Reply:
it seems that you have to modify the request url to `/user/login` at the same time, otherwise, `/apisix/admin` will appear twice under the global prefix

Reply:
oops, I will remove that.